### PR TITLE
Update financial-services-secure-collaboration.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more info link about Insider management

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3477

File: microsoft-365/solutions/financial-services-secure-collaboration.md [hunk 1]
```diff
@@ -318,7 +318,7 @@ Microsoft 365 recently launched an insider risk management solution that correla
...
 
-Insider risk management in Microsoft 365 significantly increases capabilities of organizations to monitor and investigate insider risks while allowing organizations to still meet data privacy regulations and follow established escalation paths when cases require higher-level action.
+Insider risk management in Microsoft 365 significantly increases capabilities of organizations to monitor and investigate insider risks while allowing organizations to still meet data privacy regulations and follow established escalation paths when cases require higher-level action. For more information about Insider risk management in Microsoft 365, [View Modern risk pain points and Work flow in Microsoft risk management](https://docs.microsoft.com/microsoft-365/compliance/insider-risk-management?view=o365-worldwide),
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/financial-services-secure-collaboration.md]:
```suggestion
Insider risk management in Microsoft 365 significantly increases capabilities of organizations to monitor and investigate insider risks while allowing organizations to still meet data privacy regulations and follow established escalation paths when cases require higher-level action. For more information about insider risk management in Microsoft 365, see [Modern risk pain points and Workflow in Insider risk management in Microsoft 365](https://docs.microsoft.com/microsoft-365/compliance/insider-risk-management?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@BrendaCarter Please merge this PR. Thanks

**Pull request merged**